integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/DorisContainer.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.container;
+
+import static java.lang.String.format;
+
+import com.google.common.collect.ImmutableSet;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.rnorth.ducttape.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.Network;
+
+public class DorisContainer extends BaseContainer {
+  public static final Logger LOG = LoggerFactory.getLogger(DorisContainer.class);
+
+  public static final String DEFAULT_IMAGE = System.getenv("GRAVITINO_CI_DORIS_DOCKER_IMAGE");
+  public static final String HOST_NAME = "gravitino-ci-doris";
+  public static final String USER_NAME = "root";
+  public static final String PASSWORD = "root";
+  public static final int FE_HTTP_PORT = 8030;
+  public static final int FE_MYSQL_PORT = 9030;
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  protected DorisContainer(
+      String image,
+      String hostName,
+      Set<Integer> ports,
+      Map<String, String> extraHosts,
+      Map<String, String> filesToMount,
+      Map<String, String> envVars,
+      Optional<Network> network) {
+    super(image, hostName, ports, extraHosts, filesToMount, envVars, network);
+  }
+
+  @Override
+  protected void setupContainer() {
+    super.setupContainer();
+    withLogConsumer(new PrintingContainerLog(format("%-14s| ", "DorisContainer")));
+    withStartupTimeout(Duration.ofMinutes(10));
+  }
+
+  @Override
+  public void start() {
+    super.start();
+    Preconditions.check("Doris container startup failed!", checkContainerStatus(5));
+    changePassword();
```

Comment:
If we fail to change the password, I believe we should throw an exception and stop the test.

Reply:
done. Use `Preconditions.check` instead